### PR TITLE
Update solarus ports

### PR DESCRIPTION
**Pull request by JeodC**
Fixes them for rocknix

### CODE REVIEW

File: ports/tunics/Tunics!.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/tunics/Tunics!.sh:25]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zelda-bookofmudora/The Legend of Zelda - Book of Mudora.sh [hunk 2]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zelda-bookofmudora/The Legend of Zelda - Book of Mudora.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zelda-mercurischest/The Legend of Zelda - Mercuris' Chest (DEMO).sh [hunk 3]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zelda-mercurischest/The Legend of Zelda - Mercuris' Chest (DEMO).sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zelda-mysteryofsolarusdx/The Legend of Zelda - Mystery of Solarus DX.sh [hunk 4]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zelda-mysteryofsolarusdx/The Legend of Zelda - Mystery of Solarus DX.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zelda-mysteryofsolarusxd/The Legend of Zelda - Mystery of Solarus XD.sh [hunk 5]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zelda-mysteryofsolarusxd/The Legend of Zelda - Mystery of Solarus XD.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zelda-roth-se/The Legend of Zelda - Return of the Hylian.sh [hunk 6]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zelda-roth-se/The Legend of Zelda - Return of the Hylian.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zeldaxd-mercurischess/The Legend of Zelda XD - Mercuris Chess.sh [hunk 7]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zeldaxd-mercurischess/The Legend of Zelda XD - Mercuris Chess.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zeldoschallenge/The Legend of Zelda - Zeldo's Challenge.sh [hunk 8]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zeldoschallenge/The Legend of Zelda - Zeldo's Challenge.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

File: ports/zeldoschallenge2/The Legend of Zelda - Zeldo's Challenge 2.sh [hunk 9]
```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```
Comment (kloptops) [inline ports/zeldoschallenge2/The Legend of Zelda - Zeldo's Challenge 2.sh:21]: Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```

**Review by kloptops — COMMENTED** [review]
I think the scripts are missing `$LD_LIBRARY_PATH`

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**